File: solucao-node-ts/src/ex001.ts
class ExerciseOne {
  static showAnswer() {
    const indice = 13;
    let soma = 0;
    let k = 0;

    while (k < indice) {
      k += 1;
      soma += k;
    }

    console.log(soma);
  }
}

export default ExerciseOne;

File: solucao-csharp/Exercises/Ex001.cs
namespace desafio_target.Exercises
{
    internal class ExerciseOne
    {
        public static void ShowAnswer()
        {
            int indice = 13;
            int soma = 0, k = 0;

            while (k < indice)
            {
                k += 1;
                soma += k;
            }

            Console.WriteLine(soma);
        }

    }
}
